### DISCUSSION TITLE
请问已经申请并且通过了gpt-4,切换成4还是报无法查询到模型对应的配额, 请联系管理员开通

### DISCUSSION THREAD

**Discussion by nullius007** [swuecho/chat#301] [Q&A] [answered]

**Comment by swuecho** [comment, marked as answer]
<img width="1419" alt="image" src="https://user-images.githubusercontent.com/666683/237029985-66ccb660-0687-482f-978e-15d8fb1c815a.png">

把是否单独流控关了, 默认是开的, 

或者

在模型流控那里, 给用户单独加gpt-4.

**Comment by swuecho** [comment]
前者会给所有用户开gpt4

**Reply by nullius007** [reply]
我把是否单独流控关了gpt-4就可以使用了，但是gpt-4-32k无论开关都不能使用

**Reply by swuecho** [reply]
gpt-4-32k 你有权限么? 我从来没见过谁有32k的权限. 你用官方的playground 试试 能否用

**Comment by swuecho** [comment]
后者目前有个bug需要修, 晚上下一个版本会修复.

**Reply by nullius007** [reply]
好的辛苦了

**Reply by swuecho** [reply]
你用最新的镜像 0.8.14 应该好了

如果多人共用, 尤其是公网建议用单独流控, gpt4 太贵了.

**Reply by nullius007** [reply]
多谢啦~
